src/hyperfine/types.rs

```diff
@@ -60,7 +60,7 @@ impl<'a> Command<'a> {
     pub fn new_parametrized(
         expression: &'a str,
         parameter: &'a str,
-        value: NumericType,
+        value: String,
```

Comment:
If you take a `&str` here instead of a `String`, you can avoid many of the `.clone` calls.

Reply:
I think the reason I did String was because we call `new_parameterized` for the parameter range option, we passed a `NumericType` which implements Clone. If we wanted to pass `&str` we would have to turn the `NumericType` into a `String` first and then take a reference of that, but if you look at `build_parameterized_commands`, where we call `new_parameterized`, it doesn't look like there is a very good way of doing that.